### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by OR13**
- Add proverif example
- Refactor examples and commit to source for readability
- Fix bug in the diagnostic notation for detached signatures

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -17,39 +17,38 @@ Then, install the dependencies:
 
     pip install -r requirements.txt
 
-```sh
```
Comment (SteveLasker) [inline python/README.md:20]: for `pip install -r requirements.txt`: what directory are you assuming the user is in?
Should this change to ./artiracts/requirements or should this add a cd ./artifacts
```suggestion
```sh
cd artifacts/
```

**Comment by SteveLasker** [comment]
This looks really cool, with a T1 pipe into your house. 
492M, 3.44G - gigibyte 🤷‍♂️ 
Perhaps a smaller format example?
```output
Downloading (…)nsistency_txt2img.py: 33.2kB [00:00, 273kB/s]
Downloading (…)rocessor_config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 518/518 [00:00<00:00, 1.68MB/s]
Downloading (…)_encoder/config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 610/610 [00:00<00:00, 2.89MB/s]
Downloading (…)_checker/config.json: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 726/726 [00:00<00:00, 492kB/s]
Downloading (…)cheduler_config.json: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 539/539 [00:00<00:00, 193kB/s]
Downloading (…)cial_tokens_map.json: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 133/133 [00:00<00:00, 146kB/s]
Downloading (…)okenizer_config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 765/765 [00:00<00:00, 3.54MB/s]
Downloading (…)bc04/vae/config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 651/651 [00:00<00:00, 2.84MB/s]
Downloading (…)tokenizer/merges.txt: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 525k/525k [00:00<00:00, 1.60MB/s]
Downloading (…)c04/unet/config.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1.73k/1.73k [00:00<00:00, 4.37MB/s]
Downloading (…)tokenizer/vocab.json: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1.06M/1.06M [00:00<00:00, 1.59MB/s]
Fetching 15 files:  20%|██████████████████████████████▊                                                                                                                           | 3/15 [00:17<00:04,  2.55it/s]
Fetching 15 files:  20%|██████████████████████████████▊                                                                                                                           | 3/15 [02:52<11:30, 57.50s/it]
^CTraceback (most recent call last):   0%|                                                                                                                                           | 0.00/1.73k [00:00<?, ?B/s]
  File "/home/steve/go/src/github/scitt-community/scitt-examples/python/venv/lib/python3.10/site-packages/tqdm/contrib/concurrent.py", line 51, in _executor_map           | 10.5M/492M [02:25<1:39:19, 80.8kB/s]
    return list(tqdm_class(ex.map(fn, *iterables, chunksize=chunksize), **kwargs))                                                                                       | 10.5M/3.44G [02:45<13:23:22, 71.1kB/s]
  File "/home/steve/go/src/github/scitt-community/scitt-examples/python/venv/lib/python3.10/site-packages/tqdm/std.py", line 1182, in __iter__                                | 10.5M/335M [01:35<40:39, 133kB/s]
```

File: python/README.md (hunk 1 above)
Comment (OR13) [reply python/README.md:20]: Probably I should move these instructions into the artifact itself.

Comment (OR13) [reply python/README.md:20]: I ended up moving the instructions for using the artifact into the artifact

**Comment by OR13** [comment]
You don't need to run the python code to generate an sbom for it, I have made that clearer.

**Comment by OR13** [comment]
Updated example based on feedback from OWASP CycloneDX Slack

**Review by aj-stein-nist — APPROVED** [review]

**Pull request merged**